_pages/en_US/alternate-exploits.txt

```diff
@@ -0,0 +1,61 @@
+---
+title: "Alternate Exploits"
+---
+
+{% include toc title="Table of Contents" %}
+
+3DS / 3DS XL / 2DS devices on firmware 11.17.0 do not currently have any free, software-only methods. Therefore, all of the methods here have some sort of prerequisite.
+
+### Installed eShop titles
+
+Because Nintendo eShop no longer allows purchases, it is no longer possible to purchase new titles. However, you can re-download a title if you had purchased or downloaded them before purchases stopped being possible.
+
+Any one of the following titles can be used in conjunction with Seedminer to install custom firmware:
+* Pokémon Picross (free-to-play)
+* Steel Diver: Sub Wars (free-to-play)
+* ANY [DSiWare](https://en.wikipedia.org/wiki/List_of_DSiWare_games_and_applications) title (free or paid)
+
+Continue to [Seedminer (Alternate)](seedminer-(alternate))
+{: .notice--primary}
+
+Alternatively, SmileBASIC (a paid game) can be used to get Homebrew Launcher without using Seedminer.
+
+Continue to [Homebrew Launcher (smilehax-IIe)](homebrew-launcher-(smilehax-iie))
+{: .notice--primary}
+
+
+### Retail 3DS cartridges
+
+#### ninjhax2-dx
+
+This method uses a cartridge or digital copy of the game Cubic Ninja.
+
+Continue to [Homebrew Launcher (ninjhax2-dx)](homebrew-launcher-(ninjhax2-dx))
+{: .notice--primary}
+
+#### freakyhax
+
+This method uses a cartridge of the game Freakyforms Deluxe.
+
+Continue to [Homebrew Launcher (freakyhax)](homebrew-launcher-(freakyhax))
+{: .notice--primary}
+
+
+### External hardware
+
+#### kartdlphax
+
+This method uses another 3DS with custom firmware and a digital or cartridge copy of Mario Kart 7.
+
+Continue to [Installing boot9strap (kartdlphax)](installing-boot9strap-(kartdlphax))
+{: .notice--primary}
+
+#### ntrboot
+
+This method uses a compatible DS flashcart
+
+If none of the above options are available to you, this is the cheapest available option. A preflashed ntrboot-compatible cartridge can be purchased [here](https://www.aliexpress.com/item/1005005337644588.html) for roughly $10 USD.
```

Comment:
I thought we were against putting volatile links in the guide

Reply:
i dunno, i think it makes sense to link people the actual cart that they need.

Maybe we could have this link be a redirect that we can easily change the destination of? That way the actual link doesn't cause a translation update every time it needs to be changed.

Reply:
i also just realized that the message for this sentence is incomplete lol oops

Reply:
Despite the constant negative press covfefe